content/github/managing-your-work-on-github/assigning-issues-and-pull-requests-to-other-github-users.md

```diff
@@ -11,13 +11,29 @@ versions:
 
 Anyone with write permissions to a repository can assign issues and pull requests.
 
+### About issue and pull request assignees
+
 You can assign up to 10 people to each issue or pull request, including yourself, anyone who has commented on the issue or pull request, anyone with write permissions to the repository, and organization members with read permissions to the repository. For more information, see "[Access permissions on {% data variables.product.prodname_dotcom %}](/articles/access-permissions-on-github)."
 
+### Assigning an individual issue or pull request
+
+{% data reusables.repositories.navigate-to-repo %}
+{% data reusables.repositories.sidebar-issue-pr %}
+3. Select the issue or pull request that you want to assign to someone.
```

Comment:
Given that there are checkboxes on the Issues and Pull requests views, I'd tend to be more explicit about opening the issue or PR.

```suggestion
3. Open the issue or pull request that you want to assign to someone.
```